x-pack/plugins/gis/public/components/widget_overlay/widget_overlay.js

```diff
@@ -22,6 +23,9 @@ export function WidgetOverlay() {
       <EuiFlexItem>
         <LayerControl/>
       </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <AttributionControl/>
+      </EuiFlexItem>
```

Comment:
this is just a placeholder. we might want to place the attribution readout elsewhere alltogether.